libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/helpers/WPHtmlTagHandler.java

```diff
@@ -33,27 +34,30 @@ public void handleTag(final boolean opening, final String tag, Editable output,
     }
 
     private void handleListTag(Editable output) {
-        if (mListParents.lastElement().equals("ul")) {
-            output.append("\n");
-            String[] split = output.toString().split("\n");
-            int start = 0;
-            if (split.length != 1) {
-                int lastIndex = split.length - 1;
-                start = output.length() - split[lastIndex].length() - 1;
+        int size = mListParents.size();
+        if (size > 0) {
+            if (mListParents.get(size - 1).equals("ul")) {
```

Comment:
ok so, strictly speaking, `ArrayList` allows for any of its elements to be `null`, so this would potentially have the same consequences as above. I know it seems unlikely to happen at this point in code execution, but I'd prefer us to be more conservative about it and either check for `null` before trying to access it, or moving the comparison the other way around like this: `"ul".equals(mListParents.get(size - 1))`, whatever feels more easy to read for you